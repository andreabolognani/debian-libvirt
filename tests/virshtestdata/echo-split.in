echo --split a,b,,c,,,d,,,,e,,,,,f,,,,,,e
echo --split ,a,b,,c,,,d,,,,e,,,,,f,,,,,,e,
echo --split ,,a,b,,c,,,d,,,,e,,,,,f,,,,,,e,,
echo --split ,,,a,b,,c,,,d,,,,e,,,,,f,,,,,,e,,,
echo --split ,,,,a,b,,c,,,d,,,,e,,,,,f,,,,,,e,,,,
