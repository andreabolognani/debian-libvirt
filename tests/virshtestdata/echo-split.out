a
b,c,
d,,e,,
f,,,e

a
b,c,
d,,e,,
f,,,e

,a
b,c,
d,,e,,
f,,,e,
,
a
b,c,
d,,e,,
f,,,e,

,,a
b,c,
d,,e,,
f,,,e,,
